spec/runtime/executable_spec.rb

```diff
@@ -8,142 +8,9 @@
     G
   end
 
```

Comment:
Although we could change these specs so they use `bundle binstubs` and include them, as long as they're not (too) redundant.